HACKING.md

```diff
@@ -34,10 +34,11 @@ We may then use a git submodule to import it into the Flambda backend repo.
 
 ## Code formatting for Flambda 2
 
-The CI checks that all Flambda 2 code (in `middle_end/flambda2/`) is
+The CI checks that all Flambda 2 code (in `middle_end/flambda2/`) and
+Cfg code (in `backend/cfg`) is
```

Comment:
Trailing slash after "cfg" for consistency

Reply:
fixed.